src/provider/persistence/dataLayer/IDatastore.ts

```diff
@@ -1,4 +1,4 @@
-import {Entity} from '../../../shared/Models/Entity';
+import {Entity} from '../../../Shared/Models/Entity';
```

Comment:
You lower-cased the folder, but didn't update some of the references.

Reply:
Oops. Have gone through and updated all the references now.